core/templates/dev/head/pages/exploration_player/StatsReportingService.js

```diff
@@ -84,6 +99,26 @@ oppia.factory('StatsReportingService', [
         siteAnalyticsService.registerNewCard(1);
 
         stopwatch.reset();
+        totalTime.reset();
```

Comment:
I thought totalTime was a value (e.g. number of seconds), from the name. But it looks like it's some kind of object that can be reset, so I'm confused. Maybe it needs a better name.

Clocks and watches can be reset; time is a scalar quantity.

Reply:
@seanlip Ahh. totalTime is a stopwatch object, not the time in second - better I rename it to totalTimeStopwatch or something similar.

Reply:
Renamed it for now.